contracts/apps/AppStorage.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.18;
+pragma solidity 0.4.18;
```

Comment:
Needs to be loose as it is used from `AragonApp` which needs to be loose as well